src/components/Footer.jsx

```diff
@@ -9,52 +9,38 @@ const Footer = () => {
         <div className="footerin">
           <div className="w40">
             <h3>Join Our Community</h3>
-            <a href="https://www.google.com" title="Join Now">
-              Join Now
+            <a
+              target="_blank"
+              href="https://forum.mycovidconnect.com/"
+              title="Join Now"
+            >
+              <button>Join Now</button>
             </a>
           </div>
           <div className="w30">
-            <ul>
-              <li>
-                <Link to="/">Contact Us</Link>
-              </li>
-              <li>
-                <Link to="/">FAQ</Link>
-              </li>
-            </ul>
-          </div>
-          <div className="w30">
-            <ul>
-              <li>
-                <Link to="/" title="About XCOV19">
-                  About XCOV19
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Work with Us">
-                  Work with Us
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Privacy Policy">
-                  Privacy Policy
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Terms &amp; Conditions">
-                  Terms &amp; Conditions
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Press Enquiries">
-                  Press Enquiries
-                </Link>
-              </li>
-            </ul>
+            <div className="links">
+              <a
+                target="_blank"
+                href="https://forum.mycovidconnect.com/d/2-about-xcov19-rapid-action-patient-dispatch-solution"
+              >
+                About XCOV19
+              </a>
+              <a
+                href="mailto:reachout@mycovidconnect.com"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Contact Us
+              </a>
+              <a href="#">Privacy Policy</a>
+            </div>
+            <div className="media-links">
+              <img src={Helpful} alt="helpful-logo" />
```

Comment:
@vasynizami did kemo not give out the links? Ask for relevant links as well and update